aws_doc_sdk_examples_tools/config/sdks.yaml

```diff
@@ -296,12 +296,12 @@ Python:
         uid: "boto3"
         name: "&guide-python3-api;"
     301:
-      long: "&Python301long;"
-      short: "&Python301;"
+      long: "&PythonAsynclong;"
+      short: "&PythonAsync;"
       expanded:
         long: "AWS SDK for Python Async"
```

Comment:
These names should be "Async AWS SDK for Python" and "Async SDK for Python" (in my opinion). Of course, they're likely to change later anyway, but that's my thinking on it. Other than this, these changes LGTM.